src/utils/token.ts

```diff
@@ -1,74 +1,60 @@
 import { Address, BigInt } from "@graphprotocol/graph-ts"
-import { ERC20 } from "../../generated/CollateralManager/ERC20"
-import { ERC20NameBytes } from "../../generated/CollateralManager/ERC20NameBytes"
-import { ERC20SymbolBytes } from "../../generated/CollateralManager/ERC20SymbolBytes"
+
+interface Erc20 {
+    id: string
+    name: string
+    symbol: string
+    decimals: string
+}
+
+const collateralMap: { [id: string]: Erc20 } = {
```

Comment:
how about we depend this map from `@perp/curie-deployments`? because it's the source of the truth. If `USDC` is needed in the list, we can add it to the metadata as well (but FE needs some adjustment). 

```
import mainMetadataOptimism from "@perp/curie-deployments/optimism/core/metadata.json"

const collateralMap = mainMetadataOptimism.collaterals
```

Reply:
@twxia @vinta This is just a quick patch to try out if this is the root cause of the delay. Just want to get it deployed in internal graph node and see if this fix anything. We can make it better once we identify the root cause.